### ISSUE TITLE
Can't create backup via API

### ISSUE THREAD

**Issue by Hexagon-Dev** [pterodactyl/panel#3603]
### Is there an existing issue for this?

- [X] I have searched the existing issues before opening this issue.

### Current Behavior

I make a request using php like this:

```
$headers = stream_context_create(array(
        'http' => array(
            'method' => 'POST',
            'header' => "Authorization: Bearer " . $api_key,
            'content' => "",
        ),
    ));
	
echo file_get_contents('http://mysite.com/api/client/servers/********/backups', false, $headers);
```

With error output enabled I'm getting this:
`Warning: file_get_contents(http://mysite.com/api/client/servers/********/backups): failed to open stream: HTTP request failed! HTTP/1.1 500 Internal Server Error in /var/www/html/test.php on line 17`
Also in panel logs I get some kind of database error I guess.

### Expected Behavior

I'm making a request and panel creates backup.

### Steps to Reproduce

1. Create script like that.
2. Execute it.
3. Backup wouldn't be created.

### Panel Version

1.5.1

### Wings Version

1.4.7

### Error Logs

```bash
Wings: https://ptero.co/qitapaposu
Panel: http://bin.ptdl.co/57j49
```

**Comment by DaneEveritt** [comment]
#3506 keeping this as it has details though.

**Comment by notAreYouScared** [comment]
Using his exact example, I'm able to reproduce this... 

Using an API tester works as intended... with the response below 
```json
{
  "object": "backup",
  "attributes": {
    "uuid": "06abc882-1545-4658-bf8d-edd3f32d11ef",
    "is_successful": true,
    "is_locked": false,
    "name": "Backup at 2021-09-08 02:04:51",
    "ignored_files": [],
    "checksum": null,
    "bytes": 0,
    "created_at": "2021-09-08T02:04:51-04:00",
    "completed_at": null
  }
}

```

Have you tried to use curl?

**Comment by Hexagon-Dev** [comment]
Tried to make a request using curl, got this:
`{ "errors": [ { "code": "DataValidationException", "status": "500", "detail": "Data integrity exception encountered while performing database write operation. {\"device.user_agent\":[\"The device.user agent must be a string.\"]}" } ] }`

Forgot to mention. The main reason why I decided to use this is that my schedules just don't work, when i tried to have help in discord - I got no feedback unfortunately, so hope this bug will be fixed soon..

**Comment by DaneEveritt** [comment]
Send a user agent header for now.

**Comment by DaneEveritt** [comment]
closed in 7b429831ce9b59b75d85b225a1f24ff31e3c88be
